src/templater.js

```diff
@@ -129,7 +131,7 @@
             "'": "&#39;"
           };
 
-          return replacements[tag] || tag;
+          return should_escape ? replacements[tag] || tag : tag;
```

Comment:
```suggestion
          return !!opts.autoescape ? replacements[tag] || tag : tag;
```

snake_case doesn't match the rest of the codebase. I looked at Nunjucks for inspiration and they call their configuration key `autoescape`, but I'm open to a different name